pages/common/tlmgr-conf.md

```diff
@@ -0,0 +1,32 @@
+# tlmgr conf
+
+> Manage the TeX Live configuration.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- Show the current TeX Live configuration:
+
+`tlmgr conf`
+
+- Show the current `texmf`, `tlmgr`, or `updmap` configuration:
+
+`tlmgr conf texmf|tlmgr|updmap`
```

Comment:
```suggestion
`tlmgr conf {{texmf|tlmgr|updmap}}`
```

Is this correct?

Reply:
But without braces they will not be highlighted specially. 😄 Somebody can think that it's a shell pipeline.

Reply:
New syntax is a little overkill for this. I don't think that it could be seen as a shell pipeline, because there's no spaces and the example description is pretty clear.

But I've seen that e.g. `next` and `volta` have the same issue. @navarroaxel @sbrl any thoughts on this?

Reply:
IMHO the token syntax is better here.